exercises/edward_brown/ex01_basics/src/ToLowerCase.cpp

```diff
@@ -0,0 +1,10 @@
+#include "ToLowerCase.h"
+#include <algorithm>
+#include <cctype>
+
+std::string toLowerCase(std::string MixedOrUpperCase) {
+  std::transform(MixedOrUpperCase.cbegin(), MixedOrUpperCase.cend(),
+                 MixedOrUpperCase.begin(),
+                 [](char C) -> char { return std::tolower(C); });
```

Comment:
I did this initially but then discovered the notes for [this](http://en.cppreference.com/w/cpp/string/byte/tolower). 

> Like all other functions from \<cctype\>, the behavior of std::tolower is undefined if the argument's value is neither representable as unsigned char nor equal to EOF. To use these functions safely with plain chars (or signed chars), the argument should first be converted to unsigned char